templates/web-application/package.json

```diff
@@ -25,48 +25,51 @@
     "dependencies": {
         "react": "18.2.0",
         "react-dom": "18.2.0",
-        "react-router-dom": "6.12.1"
+        "react-router-dom": "6.13.0"
     },
     "devDependencies": {
         "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
-        "@storybook/addon-essentials": "7.0.20",
-        "@storybook/addon-links": "7.0.20",
-        "@storybook/blocks": "7.0.20",
-        "@storybook/react": "7.0.20",
-        "@storybook/react-webpack5": "7.0.20",
+        "@storybook/addon-essentials": "7.0.22",
+        "@storybook/addon-links": "7.0.22",
+        "@storybook/blocks": "7.0.22",
+        "@storybook/react": "7.0.22",
+        "@storybook/react-webpack5": "7.0.22",
         "@storybook/testing-library": "0.1.0",
         "@svgr/webpack": "8.0.1",
-        "@swc/core": "1.3.63",
+        "@swc/core": "1.3.65",
         "@swc/helpers": "0.5.1",
         "@swc/jest": "0.2.26",
         "@testing-library/react": "14.0.0",
         "@types/jest": "29.5.2",
         "@types/lodash-es": "4.17.7",
         "@types/node": "18.16.18",
-        "@types/react": "18.2.12",
-        "@types/react-dom": "18.2.5",
-        "@workleap/browserslist-config": "1.2.1",
-        "@workleap/eslint-plugin": "1.8.1",
-        "@workleap/postcss-plugin": "1.2.2",
-        "@workleap/stylelint-config": "1.2.1",
-        "@workleap/typescript-configs": "2.3.1",
-        "browserslist": "4.21.8",
+        "@types/react": "18.2.13",
+        "@types/react-dom": "18.2.6",
+        "@workleap/browserslist-config": "1.2.2",
+        "@workleap/eslint-plugin": "1.8.2",
+        "@workleap/postcss-plugin": "1.2.3",
+        "@workleap/stylelint-config": "1.2.2",
+        "@workleap/typescript-configs": "2.3.2",
+        "browserslist": "4.21.9",
         "css-loader": "6.8.1",
-        "eslint": "8.42.0",
+        "eslint": "8.43.0",
         "html-webpack-plugin": "5.5.3",
         "jest": "29.5.0",
         "jest-environment-jsdom": "29.5.0",
         "jest-junit": "16.0.0",
         "lodash-es": "4.17.21",
         "msw": "1.2.2",
-        "storybook": "7.0.20",
+        "postcss": "8.4.24",
+        "postcss-load-config": "4.0.1",
```

Comment:
i'm not sure what the postcss-load-config is used for. it's the first time i see it. Pat made a sample app in the wl-config repo and it doesn't use it, so i think we could remove it. 

https://github.com/workleap/wl-web-configs/blob/main/sample/app/package.json

Reply:
This is required if I want to type correctly the `postcss.config.js`.

The other option is to ignore the `@type` specification in `postcss.config.js`.